sale_pricelist_auto_update/tests/test_sale_pricelist_auto_update.py

```diff
@@ -14,13 +15,13 @@ def setUpClass(cls):
         super(TestSalePricelistAutoUpdate, cls).setUpClass()
         cls.yearly_sales = cls.env["partner.yearly_sales"]
         range_type = cls.env["date.range.type"].create(
-            {"name": "Fiscal Year", "is_fiscal_year": True,}
+            {"name": "Fiscal Year", "is_fiscal_year": True, }
         )
         cls.date_range = cls.env["date.range"].create(
             {
                 "name": "FY2022",
                 "date_start": "2022-01-01",
-                "date_end": "2022-12-31",
+                "date_end": fields.date.today(),
```

Comment:
Why this change?

Reply:
I used the wrong approach for fixing tests case. I updated in my latest commit.